main.go

```diff
@@ -92,17 +97,60 @@ func main() {
 		err := startOperator(ctx, operatorCmd, os.Args[2:])
 		if err != nil {
 			logger.Error(err, "failed to start operator")
+			fmt.Println(fmt.Sprintf("failed to start operator: %s", err.Error()))
+			os.Exit(1)
+		}
+	case scheduledBackupCmd.Name():
+		err := startScheduledBackup(ctx, scheduledBackupCmd, os.Args[2:])
+		if err != nil {
+			logger.Error(err, "failed to create scheduled backup")
+			fmt.Println(fmt.Sprintf("failed to create scheduled backup: %s", err.Error()))
```

Comment:
🚫 **[checkstyle]** <gosimple> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline)